test/functional/test_framework/test_framework.py

```
@@ -1213,6 +1214,265 @@ def get_node(self, test: BitcoinTestFramework) -> TestNode:
             raise AssertionError(f"Node at pos {self.nodeIdx} not present, did you start the node?")
         return test.nodes[self.nodeIdx]
 
+    def register(self, node: TestNode, submit: bool, collateral_txid: Optional[str] = None, collateral_vout: Optional[int] = None,
```

Pylint (3.3.7)

[refactor] Too many arguments (17/5)

(R0913)